php/libraries/Candidate.class.inc

```diff
@@ -595,8 +605,8 @@ class Candidate
         $subprojList = array();
 
         $projID  = $this->getProjectID();
-        $query   = "SELECT SubprojectID 
-                    FROM project_subproject_rel 
+        $query   = "SELECT SubprojectID
```

Comment:
need all spaces back, otherwise, it will be like this: "SELECT SubprojectID\nFROM project_subproject_rel"